File: KoRadio/UI/ko_radio_mobile/lib/models/company_employee.dart

import 'package:json_annotation/json_annotation.dart';
import 'package:ko_radio_mobile/models/company.dart';
import 'package:ko_radio_mobile/models/user.dart';

part 'company_employee.g.dart';
@JsonSerializable()
class CompanyEmployee {
 int companyEmployeeId;
  int? userId;
  int? companyId;
  bool? isDeleted;
  bool? isApplicant;
  String? companyName;
  String? companyRoleName;
  DateTime? dateJoined;
  User? user;

  CompanyEmployee(this.companyEmployeeId);

  factory CompanyEmployee.fromJson(Map<String, dynamic> json) => _$CompanyEmployeeFromJson(json);
  Map<String, dynamic> toJson() => _$CompanyEmployeeToJson(this);
}

File: KoRadio/KoRadio.Model/DTOs/CompanyEmployee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace KoRadio.Model.DTOs
{
	public class CompanyEmployeeDto
	{
		public int CompanyEmployeeId { get; set; }
		public int UserId { get; set; }
		public int CompanyId { get; set; }
		public bool IsApplicant { get; set; }
		public bool IsDeleted { get; set; }
		public bool IsOwner { get; set; }
		public int? CompanyRoleId { get; set; }
		public DateTime DateJoined { get; set; }
		public string CompanyName { get; set; } = null!;
		public string? CompanyRoleName { get; set; }
		public CompanyDto? Company { get; set; }
		public UserManualDto User { get; set; } = null!;
	}

	public class UserManualDto
	{
		public int UserId { get; set; }
		public string FirstName { get; set; } = null!;
		public string LastName { get; set; } = null!;
		public string Email { get; set; } = null!;
		public string PhoneNumber { get; set; } = null!;
	}
	public class CompanyDto
	{
		public bool IsDeleted { get; set; }
	
	}

}
